.github/workflows/automated-tests.yaml

```diff
@@ -0,0 +1,31 @@
+name: Automated Tests
+on:
+  push:
+    branches: [master, develop]
+  pull_request:
+    branches: [master, develop]
+jobs:
+  run-lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        node-version: ["16", "17"]
```

Comment:
sorry, a little late to the PR party ;-) why run against two versions of node? normally that makes sense when running real tests but here its jsut linting and that wont change much between node versions. just makes it run longer and wastes cpu time IMO

Reply:
Yeah, that's actually right. I didn't see this.

Reply:
Then I think it would make sense to run this with the LTS version (16)

Reply:
Or even freshly released v18?

Reply:
True. Since it is only about linting, one version is enough. I would also like to try it with v18.